### PR TITLE
Add Tests into Travis CI

### PR DESCRIPTION
**Pull request by mat-rumian**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: ci/build.sh [hunk 1]
```diff
@@ -70,6 +70,30 @@ cd ../..
...
+  echo "Tracing configuration test passed"
+else
+  echo "Tracing configuration test failed"
+  exit 1
+fi
+cd ../..
+
+echo "Test terraform configuration..."
+cd ./tests/terraform
+bash run.sh
```
Comment (sumo-drosiek) [inline ci/build.sh:87]: should work without changing directory

File: ci/build.sh [hunk 2]
```diff
@@ -70,6 +70,30 @@ cd ../..
 echo "Test docker image locally..."
 ruby deploy/test/test_docker.rb
 
+echo "Test tracing configuration..."
+cd ./tests/tracing
+bash run.sh
```
Comment (sumo-drosiek) [inline ci/build.sh:75]:
```suggestion
bash ./tests/tracing/run.sh
```

File: tests/terraform/run.sh [hunk 3]
```diff
@@ -29,13 +29,16 @@ for input_file in ${INPUT_FILES}; do
...
   test_output=$(diff "${STATICS_PATH}/${output_file}" "${OUT}")
+  rm "${OUT}"
 
   if [[ -n "${test_output}" ]]; then
     echo -e "\tOutput diff (${STATICS_PATH}/${output_file}):\n${test_output}"
     test_failed "${test_name}"
+    SUCCESS=1
   else
     test_passed "${test_name}"
+    SUCCESS=0
```
Comment (sumo-drosiek) [inline tests/terraform/run.sh:40]: what if last test pass and all tests before fails?

File: deploy/helm/sumologic/upgrade-1.0.0.sh [hunk 4]
```diff
@@ -10,8 +10,8 @@ readonly PREVIOUS_VERSION=0.17
 
 readonly TEMP_FILE=upgrade-1.0.0-temp-file
 
-readonly MIN_BASH_VERSION=4.4
-readonly MIN_YQ_VERSION=3.2.1
+readonly MIN_BASH_VERSION=4.0
+readonly MIN_YQ_VERSION=3.3.0
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/upgrade-1.0.0.sh:14]: This should be 3.2.1 as before

Comment (sumo-drosiek) [reply deploy/helm/sumologic/upgrade-1.0.0.sh:14]: We can use 3.2.1 in Travis as well

**Review by sumo-drosiek — APPROVED** [review]
Please cleanup commits (squash) :)

**Review by perk-sumo — APPROVED** [review]
LGTM

**Comment by perk-sumo** [comment]
I did commits cleanup and I'm going to merge.

**Pull request merged**